docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+
+PIRL yönteminin amaç fonksiyonu aşağıdaki gibi olan NCE'yi(Gürültüye Karşıt Kestirici, Noise Contrastive Estimator) inceleyerek daha iyi anlayabiliriz.
+
+<!-- We can understand PIRL more by looking at its objective function: NCE (Noise Contrastive Estimator) as follows. -->
+
+$$
+h(v_I,v_{I^t})=\frac{\exp\big[\frac{1}{\tau}s(v_I,v_{I^t})\big]}{\exp\big[\frac{1}{\tau}s(v_I,v_{I^t})\big]+\sum_{I'\in D_{N}}\exp\big[\frac{1}{\tau}s(v_{I^t},v_{I'})\big]}
+$$
+
+$$
+L_{\text{NCE}}(I,I^t)=-\log\Big[h\Big(f(v_I),g(v_{I^t})\Big)\Big]-\sum_{I'\in D_N}\log\Big[1-h\Big(g(v_{I^t}),f(v_{I'})\Big)\Big]
+$$
+
+Burada, iki özniitelik haritası/vektörü arasındaki benzerlik metriği kosinüs benzerliği olarak tanımlanmıştır.
+
+<!-- Here we define the similarity metric between two feature maps/vectors as the cosine similarity. -->
+
+PIRL'in farkı evrişimsel öznitelik üreticisinin çıktısını doğrudan kullanmamasıdır. Aksine, evrişimsel öznitelik üreticisinden bağımsız katmanlar, $f$ ve $g$, kullanır. 
+
+
+<!-- What PIRL does differently is that it doesn't use the direct output of the convolutional feature extractor. It instead defines different _heads_ $f$ and $g$, which can be thought of as independent layers on top of the base convolutional feature extractor. -->
+
+Her şeyi bir araya getirirsek, PIRL'in NCE amaç fonksiyonu şu şekilde çalışır. Bir miniyığında(mini-batch), bir pozitif (benzer) çift ve çok sayıda negatif(benzer olmayan) çiftlere sahip olalım. Önce, dönüştürülmüş resmin öznitelik vektörü ile mini-batch'teki tüm örneklerin arasındaki benzerliği hesaplayalım. Sonra, pozitif çift için softmax-vari fonksiyonun skorunu hesaplayalım. Softmax skorunu maksimize etmemiz geri kalan tüm skorları minimize etmek demektir ki bu bizim enerji bazlı modellerde istediğimiz bir durumdur. Böylelikle, NCE kayıp fonksiyonu, benzer çiftlerin enerjilerini aşağı iterken benzer olmayan çiftlerin enerjilerini de yukarı itmemize olanak veren bir model tasarlamamızı sağlar.
+
+
+<!-- Putting everything together, PIRL's NCE objective function works as follows. In a mini-batch, we will have one positive (similar) pair and many negative (dissimilar) pairs. We then compute the similarity between the transformed image's feature vector ($I^t$) and the rest of the feature vectors in the minibatch (one positive, the rest negative). We then compute the score of a softmax-like function on the positive pair. Maximizing a softmax score means minimizing the rest of the scores, which is exactly what we want for an energy-based model. The final loss function, therefore, allows us to build a model that pushes the energy down on similar pairs while pushing it up on dissimilar pairs. -->
+
+Dr. LeCun, PIRL'ın çalışması için oldukça fazla sayıda negatif örneğe ihtiyaç uyulduğunu belirtmektedir. SGD'de mini-batchlerde negatif örneklerin çok sayıda parçasını sürekli olarak tutabilmek zor olabilir. Bü  durumu çözmek için PIRL, özel bir ön bellek yapısı kullanmaktadır.
```

Comment:
```suggestion
Dr. LeCun, PIRL'ın çalışması için oldukça fazla sayıda negatif örneğe ihtiyaç uyulduğunu belirtmektedir. SGD'de miniyığındaki negatif örneklerin çok sayıda parçasını sürekli olarak tutabilmek zor olabilir. Bü  durumu çözmek için PIRL, özel bir ön bellek yapısı kullanmaktadır.
```

I'm not sure where "parça" comes from here but I cannot offer a better sentence confidently

Reply:
I think 'negatif örneklerin büyük bir kısmını/çoğunuğunu' or 'negatif örneklerin çoğunu' is fine.